.cirrus.yml

```diff
@@ -16,6 +16,10 @@ fedora_30_task:
     matrix:
       - SETUP: setup.py develop --user
       - SETUP: -m pip install .
+  # Latest pip available avocado-71.0 contains a bug
+  # Latest pip available 69.1 contains a bug
+  allow_failures: $AVOCADO_SRC == '' || $AVOCADO_SRC == 'avocado-framework<70.0'
+  skip_notifications: $AVOCADO_SRC == '' || $AVOCADO_SRC == 'avocado-framework<70.0'
```

Comment:
For some reason the "allow_failures" still propagated to github, therefor I added the "skip_notification" which took care of it. Hopefully we'll publish the new LTS soon to get rid of one skip and new Avocado 72.0 (or minor 71.1?) to remove the other failure.

Reply:
With the newly released 69.2, there's no need anymore for those.

Reply:
OK, removing.